Makefile

```diff
@@ -34,39 +32,45 @@ docker:
 	docker cp hugo-build:/go/bin/hugo .
 	docker rm hugo-build
 
+govendor:
+	go get -u github.com/kardianos/govendor
+	go install github.com/kardianos/govendor
```

Comment:
Which is to say the `go install` line is superfluous. 

It still appears here atm: https://github.com/spf13/hugo/blob/master/Makefile#L37